include/deal.II/matrix_free/shape_info.templates.h

```diff
@@ -930,6 +931,10 @@ namespace internal
                       shape_data_on_face[0][n_dofs_1d + i])) > 1e-12)
                   element_type = tensor_symmetric;
             }
+
+
+          if (dynamic_cast<const FE_Q_iso_Q1<dim> *>(&fe))
+            element_type = tensor_symmetric_no_collocation;
```

Comment:
I would prefer if we could move this above the `if` statement for `check_1d_shapes_collocation`, because we should not even build the matrices for that case.